src/Robo/Commands/Vm/VmCommand.php

```diff
@@ -11,9 +11,6 @@
  * Defines commands in the "vm" namespace.
  */
 class VmCommand extends BltTasks {
-
-  const DRUPALVM_CONFIG_KEY = 'vm.config';
```

Comment:
more-so just wondering... do you not like the single string declaration for any reason?